docs/general/vpn-modes-exclusions.md

```diff
@@ -0,0 +1,149 @@
+
+:::
+
+## VPN modes
+
+AdGuard VPN has two modes:
+
+1. (Default) AdGuard VPN is active for all websites and apps, except for those added to the website and app exclusion lists. 
+
+2. AdGuard VPN is active *selectively*: only for the websites and apps added to the website and app exclusion lists.
+
+
+:::note
+
+You can set *different* VPN modes for apps and websites. For example, you can set AdGuard VPN to be active for all apps except for those apps added to the app exclusion list, and active only for the websites from the website exclusion list.
+
+:::
+
+## Exclusion lists
+
+An exclusion list is a list of apps or websites that AdGuard VPN treats differently than others.
+
+Exclusion lists help you fine-tune the VPN activity for specific apps or websites.
+
+
+:::note
+
+Each VPN mode has its own app exclusion list and website exclusion list. 
+In total, there are four exclusion lists:
+
+* App exclusion list [VPN active everywhere]
+* App exclusion list [VPN active selectively]
+* Website exclusion list [VPN active everywhere]
+* Website exclusion list [VPN active selectively]
+
+:::
+
+
+## Combining VPN modes and Exclusion lists
+
+We will look at the different VPN activity setups in order of decreasing VPN coverage.
+
+### AdGuard VPN is active everywhere except for apps and websites from the app and website exclusion lists
+
+VPN mode:
+* Active for all apps except app exclusions
+* Active for all websites except website exclusions
+
+Applied exclusion lists:
+* App exclusion list [VPN active everywhere]
+* Website exclusion list [VPN active everywhere]
+
+This is the default VPN mode with maximum VPN protection. If you want to exclude a website or an app from AdGuard VPN, add them to the exclusion list.
+
+**Empty exclusion lists:** Your app exclusion list and website exclusion list can be empty. If nothing is specified in the exclusion lists, AdGuard VPN will be active for all apps and websites.
+
+**Why use this mode:** You want a blanket VPN protection for everything on your device. If a specific app or website needs to work outside the VPN, add it to exclusions.
+
+:::caution
+
+If you add a browser app to the app exclusion list, AdGuard VPN won't be active for all websites opened in that browser.
+
+:::
+
+
+
+### AdGuard VPN is active (1) only for the apps from the the app exclusion list, and (2) for all websites except those from the website exclusion list
+
+VPN mode:
+* Active only for apps from app exclusions
+* Active for all websites except website exclusions
+
+Applied exclusion lists:
+* App exclusion list [VPN active selectively]
+* Website exclusion list [VPN active everywhere]
+
+
+To get into this mode, you leave AdGuard VPN active for all websites and set it to be active only for the apps from the app exclusion list. This mode provides less VPN protection coverage as you need to add apps to the app exclusion list to ensure they are protected.
```

Comment:
```suggestion
To switch to this mode, leave AdGuard VPN enabled for all websites and set it to enable only for the apps from the app exclusion list. This mode provides less VPN protection coverage because you need to add apps to the app exclusion list to ensure they are protected.
```